smarts/sstudio/genscenario.py

```diff
@@ -439,25 +439,37 @@ def _validate_entry_tactic(mission):
 
 def gen_traffic_histories(
     scenario: str,
-    histories_datasets: Sequence[str],
+    histories_datasets: Sequence[Union[types.TrafficHistoryDataset, str]],
     overwrite: bool,
 ):
     """Converts traffic history to a format that SMARTS can use.
     Args:
         scenario:
             The scenario directory
         histories_datasets:
-            A sequence of traffic history files.
+            A sequence of traffic history descriptors.
         overwrite:
             If to forcefully write over the previous existing output file
     """
-    genhistories_py = os.path.join(
-        os.path.dirname(os.path.realpath(__file__)), "genhistories.py"
-    )
     for hdsr in histories_datasets:
+        if isinstance(hdsr, types.TrafficHistoryDataset):
+            from . import genhistories
```

Comment:
You may need to add the ugly code here to fall to absolute import, if relative import throws an error here.

Reply:
oh, good point!
although it just worked in my testing, so maybe not... hmm, I don't quite understand why it sometimes works like this!

Reply:
Some users complained about relative import errors on docker. In my testing i didn't encounter any. Not sure if its based on how your repo is setup.

Reply:
I doubt that this would cause a problem since it is contained directly within the package directory tree.

Reply:
cool, updated